pvanalytics/tests/quality/test_irradiance.py

```diff
@@ -314,3 +315,72 @@ def test_daily_insolation_limits_uneven(albuquerque):
         pd.Series(True, index=ghi.index),
         check_names=False
     )
+
+
+def test_check_poa_global_limits_lorenz():
+    """Testing the function 'test_check_poa_global_limits_lorenz'"""
+
+    test_file = DATA_DIR / "lorenz_poa_test_dataset.csv"
```

Comment:
Where did this dataset come from?

Reply:
It's 5-psm data for 39.74 -105.17 (NREL's Golden campus). I got the POA values from SAM (ran a simulation with default system parameters).

Reply:
Instead of testing a "real" dataset, would it be easy/possible to craft a set of individual conditions where each combination tests a certain aspect of the function?  The `qcrad` tests are a good example of this: https://github.com/pvlib/pvanalytics/blob/main/pvanalytics/tests/quality/test_irradiance.py#L30

We're more likely to consider and explore all edge cases by assembling the inputs manually; nice clean PSM3 + SAM might not produce the exceptional conditions a QC routine is looking for.

Reply:
I agree - Actually, I didn't point that out earlier, but I did introduce some artificial issues in the data to explore different conditions. The following snapshot shows a part of the time series with the flagged time steps (The original data was all good):

![image](https://user-images.githubusercontent.com/29129716/205836931-bc51852e-5d48-441d-b334-a6c275540a79.png)

Reply:
OK, I see.  Thanks!  I still think it would be more transparent to store the problematic data like how qcrad does (in the pytest file as a dozen or rows in a numpy array instead of an external CSV) but I won't harp on it.

Reply:
I see - I will mimic what the test function for qcrad does. 👍